### DISCUSSION TITLE
Metals classifications

### DISCUSSION THREAD

**Discussion by cmutel** [sentier-dev/sentier_vocab#2] [Missing taxonomy]
### Missing taxonomy area

Steel, iron, and other metals

### Justify the need to invest resources and add complexity

There are large differences between base metals (iron, zinc, tin, etc.) themselves, and between their alloys.

I can't find good vocabularies formatted in RDF/similar, which means we would need to make up IRIs. Very happy to be corrected on this point.

Classification systems for metals are not generally public, but are instead controlled by industry associations, at least as far as I can tell. However, places like Wikipedia have [some alloy element concentrations](https://en.wikipedia.org/wiki/SAE_steel_grades). We could add other properties to these classifications later on if needed.

### Any other context

Existing metals classifications in background databases aren't great - this is an opportunity where we could make a difference.

**Comment by marc-vdm** [comment]
Lots of steel grade classification systems shared here
https://en.wikipedia.org/wiki/Steel_grades
